quinn-proto/src/crypto.rs

```diff
@@ -200,54 +236,54 @@ impl Crypto {
 
     pub fn is_1rtt(&self) -> bool {
         match *self {
-            Crypto::OneRtt(_) => true,
+            Crypto::OneRtt(_, _, _) => true,
             _ => false,
         }
     }
 
-    pub fn write_nonce(&self, state: &CryptoState, number: u64, out: &mut [u8]) {
+    pub fn write_nonce(&self, iv: &Vec<u8>, number: u64, out: &mut [u8]) {
```

Comment:
There's almost never a good reason to pass a `&Vec<u8>`; use a `&[u8]` instead (on the caller side, it will coerce automatically).